### PR TITLE
Created note deletion page UI

### LINKED ISSUE #3: Create note deletion page UI

### PR DESCRIPTION
**Pull request by KrishnapriyaSkk**
Changes:
1. Created deletion page UI.
2. Made changes based on previous PR review comments.

Fixes #3

### CODE REVIEW

File: app/javascript/src/components/Dashboard/Notes/DeleteAlert.jsx [hunk 1]
```diff
@@ -30,9 +30,7 @@ const DeleteAlert = ({
       isOpen
       isSubmitting={deleting}
       message="Are you sure you want to continue? This cannot be undone."
-      title={`Delete ${selectedNoteIds.length} ${
-        selectedNoteIds.length > 1 ? "notes" : "note"
-      }?`}
+      title={`Delete note?`}
```
Comment (yedhink) [inline app/javascript/src/components/Dashboard/Notes/DeleteAlert.jsx:33]: https://academy.bigbinary.com/learn-react/pages/miscellaneous-tips#:~:text=Rendering%20strings%20and%20blank%20spaces%20in%20JSX

File: app/javascript/src/components/Dashboard/Notes/NotesCard/Cards.jsx [hunk 2]
```diff
@@ -1,28 +1,28 @@
...
+import CardsDropdown from "./CardsDropdown";
+import { dateTimeFormatter, relativeCreationTime } from "./utils";
+
 import { USER_AVATAR_URL } from "../constants";
-import { dateTimeFormatter, relativeCreationTime } from "../utils";
 
-const { Menu, MenuItem } = Dropdown;
-const Cards = ({ item }) => (
-  <div className="m-3 w-full  border border-gray-200 p-4 shadow-sm  dark:border-gray-500 dark:bg-gray-600">
+const Cards = ({ note, setShowDeleteAlert, setSelectedNoteId }) => (
```
Comment (yedhink) [inline app/javascript/src/components/Dashboard/Notes/NotesCard/Cards.jsx:12]: The namespace has a lot of redundancies: `Notes/NotesCard/Cards.jsx`

Go through the following and make necessary changes:
- https://academy.bigbinary.com/learn-react/pages/naming-conventions-in-react
- https://bigbinary.slack.com/archives/C01T5Q4DZAR/p1663934324564009?thread_ts=1663060378.971029&cid=C01T5Q4DZAR

File: app/javascript/src/components/Dashboard/Notes/NotesCard/CardsDropdown.jsx [hunk 3]
```diff
@@ -0,0 +1,25 @@
+import React from "react";
+
+import { MenuVertical } from "neetoicons";
+import { Dropdown } from "neetoui";
+
+const { Menu, MenuItem } = Dropdown;
+
+const CardsDropdown = ({ note, setShowDeleteAlert, setSelectedNoteId }) => {
+  const OnDelete = () => {
```
Comment (yedhink) [inline app/javascript/src/components/Dashboard/Notes/NotesCard/CardsDropdown.jsx:9]: You are not paying attention to the naming. This should have been caught in your self-review process. Go through https://handbook.neetokb.com/articles/git-and-github-essentials#self-review-process

**Review by yedhink — COMMENTED** [review]
@KrishnapriyaSkk

File: app/javascript/src/components/Dashboard/Notes/NotesCard/Cards.jsx (hunk 2 above)
Comment (KrishnapriyaSkk) [reply app/javascript/src/components/Dashboard/Notes/NotesCard/Cards.jsx:12]: Hi. Will the namespace be okay if it goes `Notes/Cards/Cards.jsx` and `Notes/Cards/CardsDropdown.jsx` ? Or is a separate folder not a good practice?

Comment (yedhink) [reply app/javascript/src/components/Dashboard/Notes/NotesCard/Cards.jsx:12]: @KrishnapriyaSkk https://www.loom.com/share/4951b69878ec46069e035fe0effc166f

Comment (KrishnapriyaSkk) [reply app/javascript/src/components/Dashboard/Notes/NotesCard/Cards.jsx:12]: Yes. Thank you so much for the detailed clarification. I have now understood the same.  
Also, I couldn't use Dropdown as the component name as it conflicts with the 'Dropdown' component of neetoui. I think I should change the naming to something like 'CardDropdown' to make it specific for one single card.

Comment (yedhink) [reply app/javascript/src/components/Dashboard/Notes/NotesCard/Cards.jsx:12]: >Also, I couldn't use Dropdown as the component name as it conflicts with the 'Dropdown' component of neetoui. I think I should change the naming to something like 'CardDropdown' to make it specific for one single card.

No. That's not right. Please read: https://academy.bigbinary.com/learn-react/pages/miscellaneous-tips#:~:text=Importing%20under%20an%20alias

Comment (KrishnapriyaSkk) [reply app/javascript/src/components/Dashboard/Notes/NotesCard/Cards.jsx:12]: Okay. Thank you so much for clearing out that confusion for me.

**Pull request merged**